Events/ready.js

```diff
@@ -57,7 +58,7 @@ module.exports = async (client) => {
     }
     rolepickerCommands(client, "781296501351383050");
 
-    if (client.user.id === "740606402330099752") {
+    if (client.user.id === members.nirachanactual) {
```

Comment:
You mentioned wanting to improve the dev/prod handling. Did you have a better method in mind to replace this check?

Reply:
Beyond moving this (along with other Identifiers) into configuration, I hadn't thought of it for this section.  Might we be able to readdress later?

Reply:
As I'm concentrating only on moving magic numbers into configuration, I didn't look into this section to deeply but the more I do...

I assume the purpose of this check is to ensure Nira-Chan doesn't crash/error out when we initialize Commando here?  I notice that for the **rolepickerCommands()** call above this line, it will dump an error because that channel's not on my server.

The improvement I hope to make is to drive configuration by environment variable.  So on **NIRAENV** (making up a name here) values:

**dev** - use config.dev.json
**prod** (or empty/nonexistent by default) - use config.json

NIRAENV dev would allow us to use our own server(s). Among many settings, this will also include Commando Channel Ids so that in ready.js, we'll initialize the commands using the correct channels.

Reply:
**tl;dr** - if I can get prod(ZZ)/dev working, we shouldn't need this check. thoughts?

Reply:
My interpretation is that this check is to prevent testingNira and Nira-chan from interfering with each other when they are run on the same server, e.g. there doesn't need to be two bots attempting to handle the role picker. So the check would be modified to use your environment detection mechanism. This is fine for now.